CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+X.Y.Z Release notes
+=============================================================
+### Breaking changes
+* None.
+
+### Enchancements
+* None.
+
+### Bug fixes
+* Enabled object Ids on RN Android (#1480).
```

Comment:
Would it be more accurate to say "Fix a compilation error related to object IDs for React Native on Android"?